ionic2/app/api/api.ts

```diff
@@ -0,0 +1,134 @@
+import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+
+import { Pokemon } from './models/pokemon';
+import { PokemonIcon } from './models/pokemon-icon';
+import { Sighting } from './models/sighting';
+
+export { Pokemon, PokemonIcon, Sighting}
+
+@Injectable()
+export class Api {
```

Comment:
I just think it is very inconsistent to have the `.service.ts` suffix but no `.component.ts`. So I think we should either remove the suffixes entirely or have them for every Component and Service consistently. 
Intuitively, I would go with removing them entirely. It should be clear what is a Service and what is a Component from the context anyway.
We could also have a folder structure like so
```
/ionic2
  /components
    /searchbar
      searchbar.{ts|html|scss}
    /filter-menu
      filter-menu.{ts|html|scss}
  /models
    pokemon.ts
    sighting.ts
    pokemon-icon.ts
  /pages
    /map
      map.{ts|html|scss}
  /services
    api.ts
  /themes
    app.core.scss
```